modules/microshift-kickstart-bootc-install.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-kickstart-bootc-install_{context}"]
+= Creating a Kickstart for installing {microshift-short} on image mode for RHEL
+
+You can use the Kickstart provided with {microshift-short} for an image mode for RHEL installation.
+
+.Prerequisites
+
+* You set up the {microshift-short} Kickstart file.
+* You have the information needed to set required and optional variables.
+
+[IMPORTANT]
+====
+The subscription must include access to the `rhocp-4.x-for-rhel-9-$(uname -m)-rpms` and `fast-datapath-for-rhel-9-$(uname -m)-rpms` RPM repositories.
+====
+
+.Procedure
+
+. Set the required `BOOTC_IMAGE_URL` variable value to point to an image used in the RHEL Kickstart file installation process by running the following command:
++
+[source,terminal]
+----
+$ export BOOTC_IMAGE_URL=<myregistry>/<myorg>/<mypath>/microshift-image:tag # <1>
+----
+<1> Replace _<myregistry>_,_<myorg>_, and _<mypath>_ with your information.
+
+* The `BOOTC_IMAGE_URL` variable contains a reference to the image that is installed with the `ostreecontainer` Kickstart command. You can specify a z-stream release other than the latest by using the tag if required.
+
+. Optional. Add variables and values for registry authentication and configuration by using the following commands:
++
+.Example commands setting optional variables
++
+* Copy the `AUTH_CONFIG` variable contents to `/etc/ostree/auth.json` to authenticate access to the `BOOTC_IMAGE_URL` image by running the following command:
++
+[source,terminal]
+----
+$ export AUTH_CONFIG="$(cat ~/.quay-auth.json)" # <1>
+----
+<1> See the `5 containers-auth.jso` manual page for more information about this file format.
++
+* Copy the `REGISTRY_CONFIG` contents to `/etc/containers/registries.conf.d/999-microshift-registry.conf` to configure access to the registry containing the `BOOTC_IMAGE_URL` image by running the following command:
```

Comment:
The comment needs to be adjusted. The command sets this variable, which is then used in envsubst command below to copy the file contents to `/etc/containers/registries.conf.d/999-microshift-registry.conf`